tests/draft-next/optional/format/idn-hostname.json

```diff
@@ -331,11 +331,58 @@
                 "description": "empty string",
                 "data": "",
                 "valid": false
-            },
+            }
+        ]
+    },
+    {
+        "description": "validation of separators in internationalized host names",
+        "specification": [
+            {"rfc3490": "3.1", "quote": "Whenever dots are used as label separators, the following characters MUST be recognized as dots: U+002E (full stop), U+3002 (ideographic full stop), U+FF0E (fullwidth full stop), U+FF61(halfwidth ideographic full stop)"}
```

Comment:
Do you understand why this line wasn't annotated by the script? Are we missing a check for the `draft-next` dir maybe?

Reply:
It is explicitly excluded from the check (probably because there is no json schema specs yet for that draft)

https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/main/bin/annotate-specification-links#L82

Reply:
Aha, right, ok. I guess that makes sense for the JSON Schema spec but not others, so maybe at some point we revisit that, but thanks for reminding me!